frontend/scss/components/organisms/resources-overview.scss

```diff
@@ -0,0 +1,166 @@
+/*
+
+#############################
+### ORGANISM: code-grid ########
+#############################
+
+*/
+
+
+@import '_extends.scss';
+@import '_functions.scss';
+@import '_mixins.scss';
+@import '_variables.scss';
+
+@import 'components/atoms/_color.scss';
+@import 'components/atoms/_headline.scss';
+@import 'components/atoms/_text.scss';
+
+.#{utility('resources-overview')} {
+  grid-column: 1 / 23;
+
+}
+.#{organism('resources-overview')} {
+  position: relative;
+  margin: 11em 0 0 0;
+  padding: 3% 4.16%;
+
+  @media (min-width: 768px) {
+    padding: 3% 8.32%;
+  }
+
+  .ap-format-background {
```

Comment:
This selector either needs to be `.ap--` or one of `ap-o-`, `ap-a-`, and so on ... It'd be better to use one of the prefix helpers (`.#{utility('format-background')}`) here as well in case we'd ever change the prefix.

Reply:
changed